.gitignore

```diff
@@ -1,4 +1,3 @@
 .venv/
 *.npz
-models/
```

Comment:
Should this directory instead live within the `interface` directory? I have a lot of models in my models directory and I'm not sure if we want to include them or not.